homeassistant/components/opentherm_gw/const.py

```diff
@@ -409,25 +409,25 @@
     ],
     gw_vars.DATA_TOTAL_BURNER_STARTS: [
         None,
-        None,
+        "",
```

Comment:
This is incorrect imo. We should not set it to an empty string for this reason.


```suggestion
        "starts",
```

Reply:
This is giving me flashbacks to my high school maths teacher, but you're right, they are not actually unitless.

Reply:
That's not the impression I want to leave behind :')